CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+[![License](https://img.shields.io/github/license/CareEvolution/MyDataHelpsStarterKit)](https://github.com/CareEvolution/MyDataHelpsStarterKit)
+
+## How to contribute to MyDataHelpsUI
```

Comment:
Looks like these should be updated

Reply:
what do you think they need to be updated to? They are pointing at StarterKit?

Reply:
It says "How to contribute to MyDataHelpsUI"...presumably it should be "How to contribute to MyDataHelps Starter Kit"?

Reply:
(Here and right below)